LICENSE.note

```diff
@@ -0,0 +1,390 @@
+The implementation of vec_order() is inspired directly by data.table's
+forder() and by their contribution to R's order(). While no code from
+forder() or order() appear in the vctrs package, the implementation is
+similar enough to justify placing specific files in the vctrs package under
+the MPL-2.0 license used by data.table. Those specific files are:
+
+- order-groups.c
```

Comment:
You should include a full path here. I think you might also consider a common/unique prefix so you can say `src/order-*.c` or similar. Even better, if possible, would be to put the code in a subdirectory and then include the MPL license as `src/order/LICENSE`.